CSC-3/data/tables/cscui-sct.tbm

```diff
@@ -0,0 +1,336 @@
+#Conditional Hooks
+
+$Application: FS2_Open
+
+$On Game Init:
+[
+	AXUI = require "axui"
```

Comment:
I don't think this line is a good idea.  The global variable `AXUI` is already defined in axui-sct.tbm.  Redefining it like this seems like a recipe for unpredictable results.  I suggest either using a different variable name here or checking for `AXUI` in each function that uses it.

Reply:
This is following precedents set by other scripts i.e https://github.com/FSO-Scripters/fso-scripts/blob/master/BtASystemViewer/data/tables/sysviewer-sct.tbm
If there is an issue here it is much larger than my script.

Reply:
Oh, I see what's going on.  Since AXUI itself now loads from its own .lua file, there is no danger in reassigning the variable, because *all* AXUI variables are assigned via `require`.  So this is good to go then.